src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/Example1.java

```diff
@@ -5,8 +5,9 @@
 */
 
 package com.puppycrawl.tools.checkstyle.checks.sizes.lambdabodylength;
+
+// xdoc section -- start
 class Example1 {
-  // xdoc section -- start
   Runnable r = () -> { // OK, length is 10
```

Comment:
I was receiving the following errors so I just moved the comment above the class. This is what we want to do now anyways
```
[ERROR] [checkstyle] [ERROR] /home/stoyan/open-source/checkstyle/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/Example1.java:43:3: Comment has incorrect indentation level 2, expected is 16, indentation should be the same level as line 42. [CommentsIndentation]
[ERROR] [checkstyle] [ERROR] /home/stoyan/open-source/checkstyle/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/Example2.java:31:3: Comment has incorrect indentation level 2, expected is 16, indentation should be the same level as line 30. [CommentsIndentation]
```

Reply:
Old comment was not any different than any other that you still have below class.
I will look separately it might be some defect.

Reply:
Documentation has no information on `surrounding code` even with the link it gives that says is suppose to have detailed information. So it is all subjective if this is a bug or correct behavior.